lk2nd/panel/generated/panels.h

```diff
@@ -7,6 +7,9 @@
 #include "lk_panel_booyi_otm1287_720p_video.h"
 #include "lk_panel_cmi_nt35532_5p5_1080pxa_video.h"
 #include "lk_panel_hx8394d_720p_video.h"
+#include "lk_panel_nt35510s_fwvga_cmd_tm.h"
+#include "lk_panel_orise8012a_tmfwvga_cmd.h"
+#include "lk_panel_orise8012a_tmfwvga_cmd_bkg.h"
```

Comment:
For generated drivers, keep what you actually own. Or do you have all of them?

Reply:
downstream has this three panel, that's why I keep them here. No issue I will keep one which my device has.

Reply:
Well. I mean keep what you have and drop the rest. Drivers are large and we would add one only when we are actually using it.